sos/plugins/openstack_ironic.py

```diff
@@ -135,4 +148,24 @@ def setup(self):
             self.add_journal(units="openstack-ironic-discoverd")
             self.add_journal(units="openstack-ironic-discoverd-dnsmasq")
 
-# vim: set et ts=4 sw=4 :
+        # ironic-discoverd was renamed to ironic-inspector in Liberty
```

Comment:
Please don't delete the VIM mode line from this file.

Reply:
Weird, I'm pretty sure I did not delete it Oo

Reply:
It is deleted by this hunk, and never replaced...